README.md

```diff
@@ -28,9 +28,67 @@ pytest
 ```
 Note that tests/libp2p/test_libp2p.py contains an end-to-end messaging test between two libp2p hosts, which is the bulk of our proof of concept.
 
-# Explanation of Basic Two Node Communication
+## Feature Breakdown
+py-libp2p aims for conformity with [the standard libp2p modules](https://github.com/libp2p/libp2p/blob/master/REQUIREMENTS.md#libp2p-modules-implementations). Below is a breakdown of the modules we have developed, are developing, and may develop in the future.
 
-## Core Concepts
+> Legend: :green_apple: Done &nbsp; :lemon: In Progress &nbsp; :tomato: Missing &nbsp; :chestnut: Not planned
```

Comment:
For completeness, I think we should also include features that are not planned at the moment. We should also put :lemon: for features that have not gone through more thorough testing.

Reply:
Thanks - just added the other features and 🍋 for modules we want to test more. Please take another look.